tests/fixtures/setup/setup_hi_kumo.json

```diff
@@ -0,0 +1,356 @@
+{
+    "creationTime": 1613674982000,
+    "lastUpdateTime": 1613674982000,
+    "id": "SETUP-1234-1234-6362",
+    "location": {
+        "creationTime": 1613674982000,
+        "lastUpdateTime": 1632527573000,
+        "city": "*",
```

Comment:
Would be great to have a "real" value to know what the data look like. Same remarks for all the * values.

Reply:
Does it really matter? All * values are string values. I could replace the sensitive values with a placeholder, but since * is a string as well, I didn't bother.

Reply:
Hum OK.

Reply:
Would you like obfuscating more? So we keep the same amount of characters? I have the originals locally, so I can make changes to the regex. Currently I use the same regex as we use (in this PR) to obfuscate sensitive info in the logging.